pkg/pillar/scripts/device-steps.sh

```diff
@@ -586,6 +586,9 @@ for AGENT in $AGENTS1; do
     $BINDIR/"$AGENT" -c $CURPART &
 done
 
+# Start vaultmgr as a service
+$BINDIR/vaultmgr -c "$CURPART" runAsService &
+
```

Comment:
Since you now turned vaultmgr into a microservice I'm wondering if there's a way to get rid of initialization on line 237 in device-steps.sh to make vaultmgr self-contained.

Reply:
a) Folding the initialisation done on line 237 into vaultmgr microservice is not trivial, and needs a thought process on its own(and probably need a EVE proposal as well), may I address this in a separate PR? 

b) Updated README.md with vaultmgr

Reply:
> a) Folding the initialisation done on line 237 into vaultmgr microservice is not trivial, and needs a thought process on its own(and probably need a EVE proposal as well), may I address this in a separate PR?

Fair enough. I think this PR can go in then. That said, could you please file an EVE GitHub issue and a Tracker story to keep this on our radar?

Reply:
GitHub issue: https://github.com/lf-edge/eve/issues/327
Tracker : https://www.pivotaltracker.com/story/show/169116674